api/database/migrations/2019_10_25_142743_create_posts_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name');
+            $table->string('content');
+            $table->boolean('job_type');
```

Comment:
job_type doesn't sound right for this. is_job_ad? is_ad? is_posting? is_job?

Reply:
on it

Reply:
is_job_type or is_job?

Reply:
is_job_type is clearer, but it's a bit longer with the second underscore. is_job might be better then